doc/ref_model/chapters/chapter04.md

```diff
@@ -282,43 +282,75 @@ The idea of the Cloud Infrastructure profiles is to have a predefined set of inf
-| Management Interface                  | Specifies the bandwidth of management interface/s                                                                                                                                                       |
-
+| Root Disk                  | Specifies the size of '/' root partition. It can be considered an Ephemeral Disk |
+| CPU Parameters                  | Capability to associate virtual CPUs to the physical CPUs, and enable specific CPU architectures |
+| NUMA                  | Allow system memory allocation into cells or nodes that are associated with particular CPUs (there could be performance degradation due to an high number of requests through this shared bus) |
+| Network Parameters                  | Enable network capabilities allowing improvements on network performance (make sure that current deployment allows it, otherwise, it can lead either to a decrease on performance or instability) |
 <p align="center"><b>Table 4-12:</b> Compute Flavour Geometry Specification.</p>
 
 <a name="4.2.1.1"></a>
-#### 4.2.1.1 Predefined Compute Flavours
-The intent of the following Flavours list is to be comprehensive and yet effective to cover both IT and NFV workloads. The compute Flavours are specified relative to the “large” Flavour. The “large” Flavour configuration consists of 4 vCPUs, 8 GB of RAM and 80 GB of local disk, and the resulting virtual compute instance will have a management interface of 1 Gbps. The “medium” Flavour is half the size of a large and small is half the size of medium. The tiny Flavour is a special sized Flavour.
+#### 4.2.1.1 Predefined Compute Flavors
+
+The following tables provide a descriptive approach on how a Compute flavor can be created to suit the needs of an application, so it can be onboarded. This will minimize several challenges related to Infrastructure management:
+
+* Applications can be categorized based on their resource requirements towards cloud infrastructure
+* Additional flavors can be created which are linked to  to different VM requirements
+* Cloud infrastructure "scattering" is minimized
+* Application scaling optimization
+* Better usage of Cloud Objects (Memory;Processor;Network;Storage)
+* Applications can be categorized based on their resource requirements towards cloud infrastructure
+
+
+The intent of the following Flavors list, and respective profile is to be comprehensive and yet effective to cover both IT and NFV workloads. <br>
+
+In Kubernetes based environments these are the resource requests of the containers in the pods. To get guaranteed resources the resource requests should be set to the same values as the resource limits, to get burstable resources the resource limits should be higher than the resource requests while to get best effort resources none of resource requests of resource limits should be set.
+
+There are two flavors profiles that can be used:
+
```

Comment:
Please use the definitions consistent with [RM Chapter 02](https://github.com/cntt-n/CNTT/blob/master/doc/ref_model/chapters/chapter02.md#241-node-profiles-top-level-partitions)